src/routes/CategoriesRoute.js

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categoriesSlice';
 
 function CategoriesRoute() {
+  const dispatch = useDispatch();
+  const status = useSelector((status) => status.categories);
+
+  useEffect(() => {
+    dispatch(checkStatus());
+  }, [dispatch]);
+
   return (
     <section>
-      <button type="button">Check status</button>
+      <span>{status.toString()}</span>
+      <button type="button" onClick={() => dispatch(checkStatus())}>Check status</button>
     </section>
   );
```

Comment:
- [ ] Kindly make sure the status `UNDER CONSTRUCTION` is only being displayed if we click the `Check Status` button. When the mounted the component for the first time we must have `initialState` which is empty and will be updated to `UNDER CONSTRUCTION` once the button is clicked. At this moment the `UNDER CONSTRUCTION` status is being displayed on the first mounting of this component which is not correct. Kindly fix that.

Reply:
Thank you very much for you review.